infrastructure/src/main/java/org/corfudb/infrastructure/log/InMemoryStreamLog.java

```diff
@@ -23,7 +25,7 @@
  * Created by maithem on 7/21/16.
  */
 @Slf4j
-public class InMemoryStreamLog implements StreamLog, StreamLogWithRankedAddressSpace {
+public class InMemoryStreamLog implements StreamLog {
```

Comment:
Why delete StreamLogWithRankedAddressSpace interface?

Reply:
Make the logic static, so as to move read/write to per segment class.